packages/connection-form/src/components/connection-form.tsx

```diff
@@ -65,7 +65,7 @@ const formContainerStyles = css({
 const formContentStyles = css({
   display: 'flex',
   columnGap: spacing[3],
-  height: `calc(100vh - 310px)`,
+  maxHeight: `calc(100vh - 310px)`,
```

Comment:
We can also just make this maxHeight: 720px. In theory this calc means we could have a suuuper skinny modal, but something would have to try and make it that big to begin with.